### PR TITLE
Modify to mocap interfaces to comply with rep mocap

### PR DESCRIPTION
**Pull request by aaggj**
This Pull Request covers:

1. Mocap interfaces utilization to comply with the rep.
2. Modified the style to comply with uncrustify and cpplint.
3. Updated rolling badge in Readme.

### CODE REVIEW

File: .github/workflows/rolling.yaml [hunk 1]
```diff
@@ -30,15 +32,15 @@ jobs:
...
       - name: Install requirements
         run: pip3 install requests
       - name: Build and Test
-        uses: ros-tooling/action-ros-ci@0.2.6
+        uses: ros-tooling/action-ros-ci@0.3.15
         with:
           package-name: qualisys_driver
           target-ros2-distro: ${{ matrix.ros_distribution }}
-          vcs-repo-file-url: "https://raw.githubusercontent.com/MOCAP4ROS2-Project/mocap4ros2_qualisys/main/dependency_repos.repos"
+          vcs-repo-file-url: "https://raw.githubusercontent.com/MOCAP4ROS2-Project/mocap4ros2_qualisys/rep-mocap/dependency_repos.repos"
```
Comment (Juancams) [inline .github/workflows/rolling.yaml]: Please change this to ${GITHUB_WORKSPACE}/dependency_repos.repos

Comment (aaggj) [reply .github/workflows/rolling.yaml]: Hello! Thank you very much ☺️. The issue is that I tried using that variable, but it didn’t pass the CI. I’ll give it another try.

Comment (aaggj) [reply .github/workflows/rolling.yaml]: Done

**Comment by Juancams** [comment]
CI has disappeared, please fix it

File: .github/workflows/rolling.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    strategy:
+      matrix:
+        os: [ubuntu-24.04]
+      fail-fast: false
+    steps:
+      - name: Setup direc tories
+        run: mkdir -p ros_ws/src
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
```
Comment (fmrico) [inline .github/workflows/rolling.yaml]: I am not sure that you need to specify this. Maybe that's the reason why it is not finding the file in CI 🤔

**Review by fmrico — COMMENTED** [review]
Look this comment

File: .github/workflows/rolling.yaml (hunk 2 above)
Comment (Juancams) [reply .github/workflows/rolling.yaml]: Does v3 exist? I ask from my ignorance, since I always use v2.

And why does it create the workspace?

Comment (fmrico) [reply .github/workflows/rolling.yaml]: Yes, I have the same questions 🤔

Comment (jmguerreroh) [reply .github/workflows/rolling.yaml]: Yes, even v4: https://github.com/actions/checkout/tags

Comment (Juancams) [reply .github/workflows/rolling.yaml]: Oh, how funny, we're using a version from a year ago

Comment (aaggj) [reply .github/workflows/rolling.yaml]: > I am not sure that you need to specify this. Maybe that's the reason why it is not finding the file in CI 🤔

I have changed it, Let's see if it works.

**Comment by aaggj** [comment]
Sigue empeñado en que no encuentra el archivo de dependencias:

   vcs import: error: argument --input: can't open '/home/runner/work/mocap4ros2_qualisys/mocap4ros2_qualisys/dependency_repos.repos': [Errno 2] No such file or directory: '/home/runner/work/mocap4ros2_qualisys/mocap4ros2_qualisys/dependency_repos.repos'

**Comment by Juancams** [comment]
Uncrustify jeje 🤣

**Comment by aaggj** [comment]
> Uncrustify jeje 🤣

Uncrustified; Working, Please accept PR ;)

**Pull request merged**